### ISSUE TITLE
Feature request - disclosures on the left

### ISSUE THREAD

**Issue by caralmar** [caldwell/renderjson#33]
Thanks for this wonderful tool.  I preview large JSON docs and it would be better to have the disclosure on the left.  This way, it is offset from the rest of the keys and you can quickly visualize which keys have children by looking at the left.

For example, I work with NodeRED, and they render objects using this type of disclosure:
![image](https://user-images.githubusercontent.com/26122325/54717153-e8089980-4b2d-11e9-97a6-e30d83e8d6fc.png)

If I look at the same object section using renderjson:
![image](https://user-images.githubusercontent.com/26122325/54717201-05d5fe80-4b2e-11e9-8d4f-b714f4a866c4.png)

As you can see, it is harder to tell which keys have children.  Thanks again!

**Comment by Almenon** [comment]
You're right, disclosures on the left seems to be the better approach. It's what dev tools and vscode debugger does.
